### ISSUE TITLE
Implementing the f# d20pfsrd Reader

### ISSUE THREAD

**Issue by Freymaurer** [Redrazors/PathfinderAttackExtension#1]
Hello,

To start i think we should look into [Fable](https://fable.io/) to read in my fsharp code and make it accessible for your javascript code. Thats at least the starting point i would choose.

**Comment by Redrazors** [comment]
I tried to set up the environment to run your library last night and ran into difficulties.  Iodine says my installation of Visual Studio Code didn't find the installed build tools.  I'm out of my depth with it, as I've got absolutely no understanding of f#, net framework or any of the other build tools. Also with my other projects going on (mainly Pathbuilder on Android) I don't really have too much time to put on this.

Since it looks like the library needs to be converted into javascript to run, how about you do that your end and stick the deployed code into the repo?  I'm happy to knock up a test site somewhere to test out the javascript code, and you may be able to induce other testers with a simpler environment to run it in.

**Comment by Freymaurer** [comment]
Yeah i am currently doing some testing with javascript as i have no experience with it yet. I think i also need some time for this, but hopefully i'll find a solution soon.

**Comment by Freymaurer** [comment]
Hey so i saw your comment on the pull request and yeah i missed that. I found the necessary nuget packages you might wanna put into the references.

![Nuget dependancys](https://user-images.githubusercontent.com/39732517/56990893-958fb380-6b96-11e9-91e9-423c6b0594e6.JPG)

Also we should get another way to comunicate, as we both have halth of the knowledge we need to do this^^ Maybe you can send me your discord/slack/whatsapp, whatever via reddit.
